app/survey_config/social_survey_config.py

```diff
@@ -11,12 +12,12 @@
 class SocialSurveyConfig(
     SurveyConfig,
 ):
-    base_url: str = "https://rh.ons.gov.uk"
     survey_title: str = "ONS Social Surveys"
     footer_links: Iterable[MutableMapping] = field(default_factory=list)
     footer_legal_links: Iterable[Mapping] = field(default_factory=list)
 
     def __post_init__(self):
+        self.base_url = ACCOUNT_SERVICE_BASE_URL_SOCIAL
```

Comment:
This should be reverted. It should be on the instance attribute instead of `post_init`. This also mean you never use the base url you get passed in which is also not correct.